frontend/src/components/common/form/ParentSelect.test.tsx

```diff
@@ -9,9 +9,9 @@ import { ParentSelect } from './ParentSelect';
 import { SelectOptions } from './Select';
 
 const testOptions: SelectOptions = [
-  { value: 1, label: 'parent' },
+  { value: 1, label: 'parent', parent: '' },
```

Comment:
Was causing a console warning for the typeahead component as the `filterBy` prop always gets casted to a `string` and parent/label were passed to `filterBy`